src/rgbd/runtime.rs

```diff
@@ -12,13 +12,23 @@
 // If not, see <https://opensource.org/licenses/MIT>.
 
 use futures::future::join_all;
-use tokio::process;
+use tokio::{process, task};
+
+use clap::derive::Clap;
 
 use lnpbp::TryService;
 
 use super::Config;
 use crate::error::{BootstrapError, RuntimeError};
 
+use crate::stash::Opts as StashOpts;
+use crate::stash::main_with_config as stash_main;
+
+use crate::contracts::fungible::Opts as FungibleOpts;
+use crate::contracts::fungible::main_with_config as fungible_main;
```

Comment:
This is not critical, but it's Rust API recommendation to use namespace as a part of the name:
```suggestion
use crate::stash;
use crate::contracts::fungible;
```

and then just use `stash::Opts`, `fungible::main_with_config` etc